README.md

```diff
@@ -115,19 +115,19 @@ Get REST history:
 
 ```dart
 void getHistory([ably.RestHistoryParams params]) async {
-    // getting channel history, by passing or omitting the optional params
-    var result = await channel.history(params);
-
-    var messages = result.items; //get messages
-    var hasNextPage = result.hasNext(); //tells whether there are more results
-    if(hasNextPage){    
-      result = await result.next();  //fetches next page results
-      messages = result.items;
-    }
-    if(!hasNextPage){
-      result = await result.first();  //fetches first page results
-      messages = result.items;
-    }
+  // getting channel history, by passing or omitting the optional params
+  var result = await channel.history(params);
+
+  var messages = result.items;        // get messages
+  var hasNextPage = result.hasNext(); // tells whether there are more results
+  if(hasNextPage){    
```

Comment:
Some spacing would help readability for these `if` statements (others apply elsewhere). i.e.:

```suggestion
  if (hasNextPage) {
```

Also, FWIW, you've got excess whitespace on the end of some of these lines.

Reply:
Addressed: 593cecb3cafb89b55afd70572f9dcf02616b14ab